src/main/java/org/owasp/esapi/logging/java/JavaLogger.java

```diff
@@ -52,8 +52,7 @@ private void log(int esapiLevel, EventType type, String message, Throwable throw
 
 
     private boolean isEnabled(int esapiLevel) {
-        //Are Logger.OFF and Logger.ALL reversed?  This should be simply the less than or equal to check...
-        return (esapiLevel <= maxLogLevel && maxLogLevel != Logger.OFF) || maxLogLevel == Logger.ALL;
+        return esapiLevel >= loggingLevel;
```

Comment:
This is the logic update in each file.  All other changes are just the rename event.